lib/ui/my_workouts_page.dart

```diff
@@ -1,15 +1,44 @@
 import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/widgets/my_workouts/workout_cards.dart';
+import 'package:workoutpersonalizer_frontend/constants/styles.dart';
 
 class MyWorkoutsPage extends StatelessWidget {
+  const MyWorkoutsPage({Key? key}) : super(key: key);
+
   @override
   Widget build(BuildContext context) {
-    return Center(
-      child: Container(
-        child: const Text(
-          "My Workouts Page",
-          style: TextStyle(fontSize: 30),
-        ),
-      ),
+    return SingleChildScrollView(
+      padding: const EdgeInsets.all(32),
+      child: Column (
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          ListTile(
+            contentPadding: EdgeInsets.zero,
+            title: const Text(
+              'Workout Library',
+              style: h1
+            ),
+            trailing: IconButton(
+              icon: const Icon(
+                Icons.add_to_photos_outlined,
+                color: turquoise
+              ),
+              onPressed: () {},
+            ),
+          ),
+          const SizedBox(height: 10),
+          const WorkoutCards(),
+          const SizedBox(height: 30),
+          const ListTile(
+            contentPadding: EdgeInsets.zero,
+            title: Text(
+              'Exercise Library',
```

Comment:
Update to 'Saved Exercises'

Reply:
Is it easily possible to add one exercise to this section?

Reply:
Yes, I can add exercises pretty easily once alec's pr is merged